responsively-org/responsively-app
# Guides & Rulers

# ✨ Pull Request

### 📓 Referenced Issue 

#1149 Adds persistent guides and rulers

### ℹ️ About the PR

- Adds vertical and horizontal rulers
- Adds persistent guides which can be dragged, vertical and horizontal
- Adds keyboard short alt-r to show/hide rulers for all webviews


### 🖼️ Testing Scenarios / Screenshots

- Test with different screens resolutions
- Test with different views: single/many
- Close and reopen application after creating guides: they should remain visible
- Switch between night mode /regular mode
